src/bindgen/ir/function.rs

```diff
@@ -167,6 +168,33 @@ impl Function {
         for args in &mut self.args {
             reserved::escape(&mut args.0);
         }
+
+        // Rename pointer arguments which need to use the C-array notation
```

Comment:
It feels bad to change _types_ in a function that is about changing _names_, I wonder if we have precedents for that... If not we should change rename_for_config to something more generic I guess.

Reply:
format_for_config? IMHO, I don't think the name of this function is a problem because, from the `Rust` perspective, we are always dealing with pointers.

Reply:
I can:
- rename the function name `rename_for_config` --> `format_for_config`
- create a new function specifically for this annotation feature

Which one do you prefer?

Reply:
I think if we do this we should rename it, rather than adding an extra pass just to deal with this. But in hindsight we're already messing with the types in this pass, so maybe don't bother.

Reply:
Perfect, I'm going to leave the name as-is